install.sh

```diff
@@ -279,60 +271,58 @@ for i in $(seq 0 $((${#urls[@]} - 1))); do
   tarfile=$(basename "${url}")
   name="${tarfile%%-*}"   # extract string before first '-'
   rest="${tarfile#*-}"    # extract string after first '-'
-  version="${rest%%-chromeos*}"
-                          # extract string between first '-' and "-chromeos"
+  version="${rest%%-chromeos*}" # extract string between first '-' and "-chromeos"
 
   download_check "${name}" "${url}" "${tarfile}" "${sha256}"
   extract_install "${name}" "${tarfile}"
   update_device_json "${name}" "${version}"
 done
 
 echo_info "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
-echo -e "${GRAY}"
+
+# Work around https://github.com/chromebrew/chromebrew/issues/3305.
+sudo ldconfig &> /dev/null || true
+echo_out "\nCreating symlink to 'crew' in ${CREW_PREFIX}/bin/"
 ln -sfv "../lib/crew/bin/crew" "${CREW_PREFIX}/bin/"
-echo -e "${RESET}"
 
-echo_info "Setup and synchronize local package repo..."
-echo -e "${GRAY}"
+echo_out "Set up and synchronize local package repo..."
 
-# Remove old git config directories if they exist.
-find "${CREW_LIB_PATH}" -mindepth 1 -delete
+# Set LD_LIBRARY_PATH so crew doesn't break on i686 and
+# the mandb postinstall doesn't fail.
+echo "export LD_LIBRARY_PATH=$CREW_PREFIX/lib$LIB_SUFFIX" >> $CREW_PREFIX/etc/env.d/path
+export LD_LIBRARY_PATH=$CREW_PREFIX/lib$LIB_SUFFIX
 
-curl -L --progress-bar https://github.com/"${OWNER}"/"${REPO}"/tarball/"${BRANCH}" | tar -xz --strip-components=1 -C "${CREW_LIB_PATH}"
+# Add the CREW_PREFIX bin and musl bin directories to PATH.
+echo "export PATH=$CREW_PREFIX/bin:$CREW_PREFIX/sbin:$CREW_PREFIX/share/musl/bin:$PATH" >> $CREW_PREFIX/etc/env.d/path
+export PATH=$CREW_PREFIX/bin:$CREW_PREFIX/share/musl/bin:$PATH
 
-# Set LD_LIBRARY_PATH so crew doesn't break on i686 and
-# the mandb install doesn't fail.
-export LD_LIBRARY_PATH="${CREW_PREFIX}/lib${LIB_SUFFIX}"
+echo "export CREW_PREFIX=${CREW_PREFIX}" >> "${CREW_PREFIX}/etc/env.d/profile"
 
-# Since we just downloaded the package repo, just update package
-# compatibility information.
+# Since we downloaded the package repo, just update package compatibility information.
 crew update compatible
 
 echo_info "Installing core Chromebrew packages...\n"
+# We need these to install core.
+yes | crew install rsync
+yes | crew install pixz
 yes | crew install core
```

Comment:
These 3 lines can be combined into one with `yes | crew install rsync pixz core`.

Reply:
`rsync` and `pixz` need to be installed before `core`, otherwise the install process complains about them being missing.